yarn.lock

```diff
@@ -4325,6 +4332,11 @@ mixin-deep@^1.2.0:
   dependencies:
     minimist "0.0.8"
 
+moment@^2.22.2:
+  version "2.24.0"
+  resolved "https://registry.yarnpkg.com/moment/-/moment-2.24.0.tgz#0d055d53f5052aa653c9f6eb68bb5d12bf5c2b5b"
+  integrity sha512-bV7f+6l2QigeBBZSM/6yTNq4P2fNpSWj/0e7jQcy87A8e7o2nAfP/34/2ky5Vw4B9S446EtIhodAzkFCcR4dQg==
+
```

Comment:
Just realised MOJ frontend depends on Moment. It's used in [the timeline component](https://moj-design-system.herokuapp.com/components/timeline) but only on the server-side when Nunjucks templates are used. This is fine as it won't end up in our client-side bundle at any point.

(Context: moment.js is notoriously heavy https://bundlephobia.com/result?p=moment@2.24.0 and a very common contributor to slow frontend performance due to bloated JS bundle sizes)